src/ndt_odom_integrator.cpp

```diff
@@ -365,6 +408,47 @@ void NDTOdomIntegrator::predict_by_imu(const Eigen::Vector3d& dr)
   // ROS_INFO_STREAM("p:\n" << p_);
 }
 
+void NDTOdomIntegrator::predict_between_timestamps(const ros::Time begin_stamp, const ros::Time end_stamp)
+{
+  std::vector<nav_msgs::Odometry>::iterator itr_odom = odom_queue_.begin();
+  std::vector<sensor_msgs::Imu>::iterator itr_imu = imu_queue_.begin();
+  if (odom_queue_.size() >= 2)
+  {
+      while ((itr_odom+1) != odom_queue_.end())
+      {
+          if (begin_stamp <= itr_odom->header.stamp && itr_odom->header.stamp < end_stamp)
+          {
+              double dt = ((itr_odom+1)->header.stamp - itr_odom->header.stamp).toSec();
```

Comment:
```suggestion
              const double dt = ((itr_odom+1)->header.stamp - itr_odom->header.stamp).toSec();
```
可能な全ての変数に`const`を付けて下さい